togetherly/lib/providers/person_provider.dart

```diff
@@ -57,16 +66,24 @@ class PersonProvider with ChangeNotifier {
   }
 
   Future<void> refresh() async {
+    _ready = false;
     final familyId = _userIdentityProvider.familyId;
+    final personId = _userIdentityProvider.personId;
     if (familyId != null) {
       _parents = await _service.getParents(familyId);
       _children = await _service.getChildren(familyId);
+      _ready = true;
+      if (personId != null) {
+        _currentPerson = [...parents, ...children]
+            .singleWhere((element) => element.id == personId);
```

Comment:
This should be implemented within the getter `currentPerson`, rather than calculated and stored as part of the state.

Reply:
Done